modules/kubernetes-addons/secrets-store-csi-driver/main.tf

```diff
@@ -0,0 +1,15 @@
+resource "kubernetes_namespace" "secrets_store_csi_driver" {
```

Comment:
see other addon, should be handled by `helm_addon` I believe

Reply:
Same as above.